src/Text/Pandoc/Readers/LaTeX.hs

```diff
@@ -414,6 +416,12 @@ blockCommands = M.fromList $
   , "pagebreak"
   ]
 
+blockTextcolor :: PandocMonad m => LP m Blocks
+blockTextcolor = do
+    skipopts 
+    color <- braced
+    divWith ("",[],[("style","color: " ++ color)]) <$> grouped block <* notFollowedBy inline
+
```

Comment:
@schrieveslaach are you planning to make revisions along the lines I suggest above?